src/Symfony/Bundle/FrameworkBundle/Test/NotificationAssertionsTrait.php

```diff
@@ -22,14 +22,14 @@
  */
 trait NotificationAssertionsTrait
 {
-    public static function assertNotificationCount(int $count, string $transport = null, string $message = ''): void
+    public static function assertNotificationCount(int $count, string $transportName = null, string $message = ''): void
```

Comment:
Could it break the usage of named arguments? https://www.php.net/manual/en/functions.arguments.php#functions.named-arguments

I wonder if it's applicable here.

Reply:
Yes, the BC policy allows this. Also, the original feature was merged a few days ago in Symfony 6.2, which hasn't been released yet.